### ISSUE TITLE
User Profile storage on S3 failed

### ISSUE THREAD

**Issue by samuzzal-choudhury** [fabric8-analytics/fabric8-analytics-worker#141]
The bookkeepertask fails when tries to store user profile on S3 with the following error -

```
  File "/usr/lib/python3.4/site-packages/botocore/validate.py", line 291, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter ServerSideEncryption, value: True, type: <class 'bool'>, valid types: <class 'str'>
```
### Reason:
This happens because of the wrong encryption method set in the nodes.yml file for S3UserProfileStore adapter.

### Remedy
Make S3UserProfileStore encryption false which is in sync with other S3 adapters.

**Comment by samuzzal-choudhury** [comment]
aws:kms is the appropriate choice for encryption

**Comment by jpopelka** [comment]
Fixed by #142
